scr/hw6/Main.java

```diff
@@ -0,0 +1,32 @@
+package hw6;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner s = new Scanner(System.in);
```

Comment:
Where do you use variable `s`?

Reply:
I tried several validation options, and one of them was data entry for an array